style.css

```diff
@@ -61,9 +76,162 @@ article p {
   display: inline-flex;
   justify-content: space-between;
   min-width: 25vh;
+  margin: 0.1px;
+  padding: 0.1px;
+}
+
+.card-list li {
+  list-style-type: none;
+  padding: 5px 10px 5px 10px;
+  border-radius: 6px;
+  color: #6070ff;
+  background: #ebebff;
+  margin: 5px;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.about-list > li {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-weight: 300;
+  font-size: x-large;
+  color: #172b4d;
+}
+
+.lang-drop li {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.lang-drop > li {
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin-bottom: 10px;
+  padding: 5px 0.1px 5px 10px;
 }
 
 .social-links ul li {
   list-style-type: none;
   width: 0%;
 }
+
+.card-image img {
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.card-content article {
+  padding-left: 10px;
+}
+
+.card-time p {
+  font-family: 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', sans-serif;
+  font-size: smaller;
+  font-weight: 600;
+  color: #7a869a;
+}
+
+.about-content p {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-weight: 100;
+  font-size: large;
+  color: #172b4d;
+}
+
+.card-content article p {
+  font-family: 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', sans-serif;
+  font-weight: 100;
+  font-size: large;
+}
+
+.card-time .darken {
+  color: #344563;
+}
+
+.content article #invisible {
+  visibility: hidden;
+}
+
+.card-image {
+  width: auto;
+}
+
+.card-time {
+  display: flex;
+}
+
+.card-list {
+  list-style-type: none;
+  float: right;
+  padding: 0.1px;
+  margin: 0.1px;
+  display: flex;
+  justify-content: flex-start;
+  margin-bottom: 20px;
+}
+
+.card-button {
+  padding: 13px 5px 13px 5px;
+  width: 130px;
+  height: 43px;
+  border-radius: 8px;
+  color: #396df2;
+  border: 1px #396df2 solid;
+  background: white;
+  font-size: 18px;
+  font-weight: 250;
+  transition: 1s;
+  margin-bottom: 10px;
+}
+
+.card-button:hover {
+  background: lightgray;
+  cursor: pointer;
+}
+
+.about-content {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.about-content h1 {
+  font-size: xx-large;
+  font-weight: 800;
+  color: #172b4d;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.about-button {
+  margin-top: 10px;
+  padding: 13px 5px 13px 5px;
+  width: 150px;
+  height: 43px;
+  border-radius: 8px;
+  color: #396df2;
+  border: 1px #396df2 solid;
+  background: white;
+  font-size: 18px;
+  font-weight: 250;
+  transition: 1s;
+  margin-bottom: 10px;
+}
+
+.about-sh-p {
+  font-family: sans-serif;
+  color: #7f8cff;
+  font-weight: 300;
+}
+
+.about-button:hover {
+  background: lightgray;
+  cursor: pointer;
+}
```

Comment:
- [ ] [REQUIRED] The "About myself" section should be the margins (top, bottom, left, right) defined in the design. But your design is so different, please make changes!

![image](https://user-images.githubusercontent.com/63915024/170587281-43da4a69-9919-45e3-b9ee-283bc38fe49b.png)

Reply:
done